src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+import lotto.domain.vo.LottoNumber
+
+class WinningLotto {
+    private val winningLotto: Lotto
+
+    constructor(winningNumbers: List<Int>) {
+        winningLotto = Lotto(winningNumbers.map(::LottoNumber).toSet())
+    }
```

Comment:
```suggestion
class WinningLotto(winningNumbers: List<Int>) {
    private val winningLotto = Lotto(
        winningNumbers.map(::LottoNumber)
            .toSet()
    )
```
부생성자가 아닌, 주 생성자를 활용해보면 어떨까요?

Reply:
좋은 의견입니다 😄